onboarding/general-member.md

```diff
@@ -0,0 +1,68 @@
+# General Member onboarding
+
+Welcome to the TODO Group!
+
+This guide should provide your company representatives helpful onboarding
+resources once your [TODO General Membership Application](https://joinnow.todogroup.org/)
+has been validated and processed.
+
+An issue should have been created to process your company's membership (via this [template](/.github/ISSUE_TEMPLATE/onboarding-general-member.md)).
+
+If you run into any issues during the onboarding process, please reach out to the TODO Group Program Manager ([@anajsana][todo-pm]).
+
+- [Communication channels](#communication-channels)
+  - [Registration required](#registration-required)
+  - [Social media](#social-media)
+  - [GitHub](#github)
+  - [Private channels](#private-channels)
+- [Meetings and Calls](#meetings-and-calls)
+  - [Public calls](#public-calls)
+  - [Private calls](#private-calls)
+- [Resources and initiatives](#resources-and-initiatives)
+
+## Communication channels
+
+<!--
+TODO: Publish this info on the website to de-duplicate content
+-->
+
+### Registration required
+
```

Comment:
I'd add an intro description. Something like:

"Public communication channels that need registration wihtin third-party platforms to either see content or interact with content (Slack, LF Community Platform or GitHub)"